src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,26 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import './TodoInfo.scss';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })
+      }
```

Comment:
```suggestion
          <article
            className={cn('TodoInfo', {
              'TodoInfo--completed': headers,
            })}
          >
```

Reply:
You've got an eagle eye, Yurii. Thanks!
But what about 'headers', what should it be?..